### DISCUSSION TITLE
How I solved the problem of verifying student documents? 🎉

### DISCUSSION THREAD

**Discussion by eneskutlay** [community/community#81605] [GitHub Education]
### Hi everyone,

Hey there! _I spent a whole day trying to renew my expired GitHub student membership_, and finally stumbled upon a neat trick that might help some of you out there.

**If you're having trouble getting your document rejected, here's what worked for me:**

- Use a mobile device to upload the document, not a computer.
- Instead of selecting the document image from your photo gallery, try using the camera to take a fresh picture of the document and upload it.

I had such a frustrating time when GitHub kept rejecting my official student document for no clear reason. But by using this method, I was able to resolve it without any problems this time. Hopefully it can save some people the headache I went through. Sorry for the harsh experience - I know how frustrating it can be.

**Comment by devmuhib009** [comment]
Thank you for sharing this tip! I had a similar issue and this workaround worked wonders. Your advice might save others from the same frustration. Appreciate it!

**Comment by tobycm** [comment]
What also worked for me was to include your finger in the image, but don't cover any information (lol)
